### DISCUSSION TITLE
Introduction: Matt Osborn

### DISCUSSION THREAD

**Discussion by mattosborn** [life-itself/community#421] [Introductions]
Hi all,

I'm currently at the Praxis Hub in Bergerac between the two residencies I'm attending, and I landed here on GitHub after many chats with people here at the hub during the last residency (thank you @Liyubov for the invitation).

Firstly to say, I love the project. It's bringing together many things that I deeply care about and appreciate, so I'm grateful for stumbling upon it and grateful to all involved. ❤️

My professional background is software engineering, mostly full-stack development, though I've been coding for about 20 years now & have strayed into other areas here and there too, either for work or fun or curiosity.

I have a couple of on-going roles at the moment – one as a tech lead for a [non-profit foundation](https://www.fundaciovit.org/) focused on open data & dataviz, and the other teaching engineering content for a [course around the same topic](http://www.mastervisualtoolsudg.com/). I've also been working in the web3 space & am currently working on a research project looking at on-chain governance design. Loosely related to this I'll be joining the [Roote fellowship](https://www.roote.co/fellowship) for the RF7 cohort starting this summer.

Outside of work & digital life I've lived pretty much exclusively in co-living communities (I guess since birth even, if you can count a large family home in that category??). In 2021 I helped to cofound a small co-living project in rural Wales, and I'm currently involved in the early stages of a new co-housing community in the Devon / Totnes area in the UK.

Looking forward to meeting you all and seeing how I might be able to get involved.

**Comment by nathenf** [comment]
Welcome Matt! Looking forward to connecting. If you haven't joined yet, be sure to join our community chat as well on Whatsapp and come to a community call https://chat.whatsapp.com/JNJCTZugNQn1fq89xbHtfA

**Reply by Liyubov** [reply]
yes we are really looking forward to meet and discuss with you more @mattosborn and see you Friday!

**Comment by rufuspollock** [comment]
@mattosborn a massive welcome and great to have you involved 🎉

**Comment by baouroux** [comment]
Look forward to meeting Matt, thanks for the introduction!
